scripts/activate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# call connectedhomeip setup script
+./third_party/chip/repo/scripts/activate.sh
```

Comment:
```suggestion
source ./third_party/chip/repo/scripts/activate.sh
```
We have to source the script instead of running it so it can set environment variables in the current session